src/pages/ManageComputeHost/DeactivateComputeHost.js

```diff
@@ -0,0 +1,78 @@
+// (c) Copyright 2019 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import DisableComputeServiceNetwork from '../ReplaceServer/DisableComputeServiceNetwork';
+import { translate } from '../../localization/localize';
+
+const MIGRATE_INSTANCES = 'migrate_instances',
+  NOVA_STOP = 'nova-stop',
+  DISABLE_COMPUTE_SERVICE = 'nova_deactivate';
+
+class DeactivateComputeHost extends DisableComputeServiceNetwork {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ...this.state,
+      heading: translate('server.deactivate.text', this.props.operationProps.oldServer.hostname)
```

Comment:
Tried a couple of times. this.props will cause errors for me. Remove this. works fine